bootstrap.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "Boostrapping!"
+
+mkdir -p deps/ && cd deps/
+
+if [[ ! -d ./mapnik-vector-tile ]]; then
+    echo "Cloning mapnik-vector-tile"
+    git clone git@github.com:mapbox/mapnik-vector-tile.git  
+fi
+
+echo "Building mapnik-vector-tile"
+
+cd mapnik-vector-tile 
+git checkout tags/v0.11.0
```

Comment:
This is the last mapnik-vector-tile version that has a compatible boost version with Ubuntu Precise.